src/components/Board.js

```diff
@@ -12,22 +12,90 @@ class Board extends Component {
     super();
 
     this.state = {
-      cards: [],
+      cards: []
     };
   }
+  componentDidMount() {
+    axios.get('https://inspiration-board.herokuapp.com/boards/Semret/cards')
```

Comment:
There might be a good way to use the props `url` and `boardName` that the `App` component passes in for this (you'd need to change the `boardName` that `App` defines)